lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -207,43 +207,50 @@ redistribute_policy:
     config_set_append: ' <state> redistribute <protocol> route-policy <policy>'
 
 #config in vrf context
+# TODO: fix these for XR
 route_target_both_auto:
```

Comment:
This was just an easy way to "comment them out" without making other changes.  Marking them excluded for XR makes the getters return nil and causes a nil error in some of the code.  I didn't want to start changing code that is currently being worked on in US59615, so I just used this work-around.